srcpkgs/latte-dock/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'latte-dock'
 pkgname=latte-dock
-version=0.10.4
+version=0.10.7
```

Comment:
.8 is out

Reply:
Oh! Thanks for the heads up! I'll take a look at that asap

No idea how I can update this pull request though, I am relatively new to git.

Reply:
Updated, and changed the relevant parts of PR accordingly.